src/core/EdFi.DataManagementService.Core/Handler/DeleteByIdHandler.cs

```diff
@@ -53,8 +53,8 @@ DeleteFailureReference failure
                     Headers: []
                 ),
             DeleteFailureWriteConflict => new FrontendResponse(StatusCode: 409, Body: null, Headers: []),
-            UnknownFailure failure => new(StatusCode: 500, Body: failure.FailureMessage, Headers: []),
-            _ => new(StatusCode: 500, Body: "Unknown DeleteResult", Headers: [])
+            UnknownFailure failure => new(StatusCode: 500, Body: failure.FailureMessage.ToJsonError(), Headers: []),
+            _ => new(StatusCode: 500, Body: "Unknown DeleteResult".ToJsonError(), Headers: [])
```

Comment:
Previously, this was generating a 500 with a plain string body instead of a JSON body.